docs/modules/ROOT/pages/scaling-upgrading.adoc

```diff
@@ -35,3 +37,18 @@ Assuming the following is the existing configuration:
 include::ROOT:example$/hazelcast.yaml[]
 ----
 
+There are two system properties worth mentioning related to upgrade:
+
+- During rolling upgrade cluster is shutdown gracefully to prevent data loss. You might want to configure link:https://docs.hazelcast.com/hazelcast/5.3/system-properties#hazelcast.graceful.shutdown.max.wait[hazelcast.graceful.shutdown.max.wait system property].
```

Comment:
```suggestion
- During a rolling upgrade, the cluster is shutdown gracefully to prevent data loss. You can configure this using the link:https://docs.hazelcast.com/hazelcast/5.3/system-properties#hazelcast.graceful.shutdown.max.wait[hazelcast.graceful.shutdown.max.wait system property].
```